src/transformers/models/speecht5/number_normalizer.py

```diff
@@ -0,0 +1,186 @@
+# coding=utf-8
+# Copyright 2023 The Facebook Inc. and The HuggingFace Inc. team. All rights reserved.
```

Comment:
Is speecht5 not from microsoft?

Reply:
Yes, Apologies, I copy-pasted the comments at the top from another file and forgot to change it.